flintrock/flintrock.py

```diff
@@ -297,6 +305,21 @@ def launch(
         raise UnsupportedProviderError(provider)
 
 
+def get_last_commit_sha(git_repository: str):
+    """
+    Retrieve the last commit's SHA from a repository hosted on Github.
+    """
+    parsed_url = urllib.parse.urlparse(git_repository)
+    if parsed_url.netloc != "github.com":
+        raise UsageError('Error: Retrieving the last commit\'s SHA only works with '
```

Comment:
Do you mean that the `UsageError`s should be double quoted string as well?
Because throughout the file they're single quoted.